LeaderboardBackend/Services/Impl/LeaderboardService.cs

```diff
@@ -3,26 +3,18 @@
 
 namespace LeaderboardBackend.Services;
 
-public class LeaderboardService : ILeaderboardService
+public class LeaderboardService(ApplicationContext applicationContext) : ILeaderboardService
 {
-    private readonly ApplicationContext _applicationContext;
+    private readonly ApplicationContext _applicationContext = applicationContext;
```

Comment:
Don't need to store the parameter when using a primary constructor. See https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/instance-constructors#primary-constructors.